src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -830,7 +830,7 @@ RCLConsensus::timerEntry(NetClock::time_point const& now)
     catch (SHAMapMissingNode const& mn)
     {
         // This should never happen
-        JLOG(j_.error()) << "Missing node during consensus process " << mn;
+        JLOG(j_.error()) << "During consensus process: " << mn.what();
```

Comment:
Don't need the `.what()` (here and below) if you're still defining the stream operator (it still used in other places in the code).

Reply:
I'm thinking of just removing the `<<` and printing `what()` instead. Not sure if that's better.

Reply:
I look at the stream `operator<<` as the C++ version of `__str__` in Python or `Show` in Haskell, generally considered good practice, so I like it.

Reply:
Hmmm. That seems reasonable.